common/changes/office-ui-fabric-react/buttonTypo_2019-04-03-00-05.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Button: Adding missing parenthesis.",
```

Comment:
nit: while this is what you did, it doesn't actually provide useful info in release notes :P